docs/conceptual/gpu-arch/mi200-performance-counters.md

```diff
@@ -1,455 +1,591 @@
-| `sqc_tc_inst_req`              | Req    | Number of instruction cache line requests to L2 cache             |
-| `sqc_tc_data_read_req`         | Req    | Number of data read requests to the L2 cache                      |
-| `sqc_tc_data_write_req[∗]`     | Req    | Number of data write requests to the L2 cache                     |
-| `sqc_tc_data_atomic_req[∗]`    | Req    | Number of data atomic requests to the L2 cache                    |
-| `sqc_tc_stall[∗]`              | Cycles | Number of cycles while the valid requests to L2 cache are stalled |
+| Hardware counter | Unit   | Definition |
+| :---------------------| :------| :----------|
+| `SQC_ICACHE_REQ` | Req | Number of `L1I` cache requests |
+| `SQC_ICACHE_HITS` | Count | Number of `L1I` cache hits |
+| `SQC_ICACHE_MISSES` | Count | Number of non-duplicate `L1I` cache misses including uncached requests |
+| `SQC_ICACHE_MISSES_DUPLICATE`  | Count  | Number of duplicate `L1I` cache misses where the previous lookup miss on the same cache line is not fulfilled yet |
+| `SQC_DCACHE_REQ` | Req | Number of `sL1D` cache requests |
+| `SQC_DCACHE_INPUT_VALID_READYB` | Cycles | Number of cycles while SQ input is valid but sL1D cache is not ready |
+| `SQC_DCACHE_HITS` | Count | Number of `sL1D` cache hits |
+| `SQC_DCACHE_MISSES` | Count | Number of non-duplicate `sL1D` cache misses including uncached requests |
+| `SQC_DCACHE_MISSES_DUPLICATE` | Count  | Number of duplicate `sL1D` cache misses |
+| `SQC_DCACHE_REQ_READ_1` | Req | Number of constant cache read requests in a single DW |
+| `SQC_DCACHE_REQ_READ_2` | Req | Number of constant cache read requests in two DW |
+| `SQC_DCACHE_REQ_READ_4` | Req | Number of constant cache read requests in four DW |
+| `SQC_DCACHE_REQ_READ_8` | Req | Number of constant cache read requests in eight DW |
+| `SQC_DCACHE_REQ_READ_16` | Req | Number of constant cache read requests in 16 DW |
+| `SQC_DCACHE_ATOMIC[∗]` | Req | Number of atomic requests |
+| `SQC_TC_REQ` | Req | Number of TC requests that were issued by instruction and constant caches |
+| `SQC_TC_INST_REQ` | Req | Number of instruction requests to the L2 cache |
+| `SQC_TC_DATA_READ_REQ` | Req | Number of data Read requests to the L2 cache |
+| `SQC_TC_DATA_WRITE_REQ[∗]` | Req | Number of data write requests to the L2 cache |
+| `SQC_TC_DATA_ATOMIC_REQ[∗]` | Req | Number of data atomic requests to the L2 cache |
+| `SQC_TC_STALL[∗]` | Cycles | Number of cycles while the valid requests to the L2 cache are stalled |
 
 ### Vector L1 cache subsystem
 
-The vector L1 cache subsystem counters are further classified into texture addressing unit, texture data unit, vector L1D cache, and texture cache arbiter.
+The vector L1 cache subsystem counters are further classified into texture addressing unit, texture data
+unit, vector L1D cache, and texture cache arbiter.
 
 #### Texture addressing unit
```

Comment:
@SwRaw this is an empty heading; We should add content here or remove the following heading.